__tests__/redis.test.js

```diff
@@ -0,0 +1,35 @@
+const axios = require('axios');
+const list = require('../database/list.js');
+const redis = require('redis');
+
+const redisClient = redis.createClient();
+
+describe('redis cache should', () => {
+  test('contain the same data as in mongoDB for the same restaurant id', (done) => {
+    axios('http://localhost:3002/api/restaurants/7/gallery')
+      .then(() => {
+        redisClient.get('7', (err, redisRes) => {
+          if (err) {
+            console.log('document not in redis!');
+            expect(err).not.toBe(err);
+            done();
+          } else {
+            list.find({ place_id: '7' })
+              .then(photos => {
+                expect(redisRes).toBe(JSON.stringify(photos[0]));
+                done();
+              })
+              .catch((err) => {
+                console.log('document not in mongoDB!');
+                expect(err).not.toBe(err);
+                done();
+              });
+          }
+        });
+      })
+      .catch((err) => {
+        console.log('axios error: ', err);
+        done();
+      });
+  });
+});
```

Comment:
I think testing this is good, but when there is extra time perhaps you could re-examine that 1) currently the test is dependent on redis and the main database being configured correctly with pre-filled data outside of the test and that 2) it seems to be testing whether redis / main database works as advertised. Perhaps it would be better to test the server/index.js's try-cache-then-main-database logic by mocking out the two databases.